include/eve/module/algo/algo/remove.hpp

```diff
@@ -77,6 +77,8 @@ namespace eve::algo
   //!
   //!   TODO: docs
   //!
+  //!   @see keep_if
+  //!
```

Comment:
why here you missing all the others?

Reply:
forgot, i will add them

Reply:
actually i have only added closely related functions to these sections, as to not overcrowd it. but do you think it would be better if all functions of the family linked to the entire family? maybe a dedicated page even?

Reply:
I think they should ber listed here. An extra page - meh.